### ISSUE TITLE
Looking for maintainers

### ISSUE THREAD

**Issue by xabre** [xabre/xamarin-bluetooth-le#274]
If you are interested in being a maintainer let's start a conversation here :)

**Comment by smsissuechecker** [comment]
Hi @xabre,

I'm the friendly issue checker.
It seems like (100.00 %) you haven't used our issue template :cry: I think it is very frustrating for the repository owners, if you ignore them.

If you think it's fine to make an exception, just ignore this message.
**But if you think it was a mistake to delete the template, please close the issue and create a new one.**

Thanks!

**Comment by flyingxu** [comment]
Hi, I'd love to be one of them.

My day job is a Xamarin app using Bluetooth, I started using this package more than a year ago, call the APIs almost every day (but it's stable now so I don't debug into it often).
I tried porting the code to UWP (Some other's PR get merged but I got familiar with the code structures). I am planning to add MacOS support as well because it's needed by my day job.
I'm also a contributor of MvvmCross (accepted as a team member). This package introduced MvvmCross to me and then I spent more time there and use MvvmCross every day.

I'm not very active in the PRs here because I think it's quite stable now, at least for Android and iOS. 

This is a good package and it will help others if it gets maintained and adds more platform support.

**Comment by xabre** [comment]
Hi @flyingxu, 
thank you for stepping up. We need some help with sorting through the open issues, maybe doing a few releases with the easy fixes. Great idea with the MacOS port, we have been wanting to add it ever since XF realease MacOS support which makes the sample app usable....but sadly not enough time. Doing the MacOS/UWP ports I think is a great way to get onboarded.

**Comment by xabre** [comment]
I have just sent you an invite :)

**Comment by flyingxu** [comment]
Thanks, I'll begin with the issues.

**Comment by bed007** [comment]
Hi @xabre ,

I have been also using plugin-ble for quite a long time now and on several projects. I've also already contributed (once) to the project. I might have some time to contribute to the project. If you are interested in having extra hands, I would be happy to help!

**Comment by jeevaeastpoint** [comment]
I would like to contribute as well

**Comment by lucafabbri** [comment]
Glad to help!

**Comment by bed007** [comment]
@xabre looks like there is few people that wants to step in this project. Would you consider adding more collaborators to the project ?

**Comment by xabre** [comment]
@all Thanks for the support, from my point of view the more the merrier :)

**Comment by xabre** [comment]
@bed007 I've just added you as a collaborator :) Glad to have you onboard

**Comment by erdmenchen** [comment]
Since this project is not actively maintained, do you recommend any other xamarin forms ble plugin?

**Comment by xabre** [comment]
@erdmenchen well this is not the right place for your question, if you would be interested in becoming a colaborator and 'actively' maintain the project then this is the right place for you :) and we welcome you with open arms, if not try stackoverflow.

**Comment by pullelamurthy** [comment]
I can also help. I have been using this plugin for a long time and want to contribute to make this better. Please inform if there is anything I can do to help.

**Comment by centrolutions** [comment]
How can I help get UWP support off the ground? Is there a starting point anywhere (i.e. the code that was tried and shelved)? I'd be happy to contribute to it since we have an existing project I'm interested in adding UWP support in.

**Comment by flyingxu** [comment]
@centrolutions for UWP, you can see there is a branch of 'uwp_creators_update' at https://github.com/xabre/xamarin-bluetooth-le/branches

I tested and it work. The next step is to merge it into main branch

**Comment by bugail** [comment]
@xabre I'd like to become a collaborator. Been using a plugin for awhile and have had great success with it. Been seeing a few issues with Android 8/9 and would love to help fix them :)

**Comment by tfalasco** [comment]
Hello,
I would like to help maintain this project. A few years ago I wrote my own cross-platform BLE implementation for a Xamarin Forms app I wrote because I needed to support UWP, Android, and iOS. At the time this plugin didn't support UWP (plus I wanted to learn).
I like the work you're doing here and want to see it move forward. I think I can help out. Let me know if you're still looking for contributors. 
Thanks, 
Ted

**Comment by xabre** [comment]
@bugail @tfalasco Nice to have you onboard.

**Comment by JanNepras** [comment]
@xabre @flyingxu We are using your plugin on Android and iOS. We also need UWP and I would like to get it done. Could you add me as a contributor?

**Comment by slimjim10** [comment]
I will be implementing Cancellation token for some calls which are currently not implemented, for example GetServiceAsync, GetCharacteristicAsync, ReadSync and WriteAsync.

I think this would be good to also merge into this project. How should I continue with this? Just do a pull request, or do you want to add me as a contributor?

**Comment by jpark318** [comment]
@xabre I am currently using this wonderful package for my research team. Will be glad to help to maintain this plugin further, and I will also contribute to having ble part done for uwp as well! Could you add me as a contributor?

**Comment by JanNepras** [comment]
@jpark318 I am going to work on the uwp probably next week. Could we work on this together?

**Comment by jpark318** [comment]
> @jpark318 I am going to work on the uwp probably next week. Could we work on this together?

@JanNepras Yes, is there any documentation that would help me understand current progress on UWP? or should I just go through the code existing in uwp_creators_update branch??

**Comment by MKuckert** [comment]
@xabre any chance to get on board also?

**Comment by Thorinos** [comment]
I would like to be a collaborator too! I've send a pull request a few weeks ago and I would like to fix some issuse with the app demo. Let me know how I can help!
